packages/player/src/styles.scss

```diff
@@ -15,6 +15,127 @@ $states: ('initial' 'loading' 'ready' 'error');
   display: block;
 }
 
+.controls {
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between; /* Align button to the left and timeline to the right */
```

Comment:
We could also remove all the comments here IMO . The styles are very self-explanatory. If you want to leave in some that's ok too, just an idea.

Reply:
no this is all ChatGPT, I don't know CSS